slang-plugin/src/main/java/org/sonarsource/slang/plugin/RulesDefinitionUtils.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * SonarSource SLang
+ * Copyright (C) 2009-2018 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonarsource.slang.plugin;
+
+import java.lang.reflect.Field;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.sonar.api.server.rule.RulesDefinition;
+import org.sonar.api.utils.AnnotationUtils;
+import org.sonar.check.RuleProperty;
+import org.sonarsource.slang.checks.utils.Language;
+import org.sonarsource.slang.checks.utils.PropertyDefaultValue;
+import org.sonarsource.slang.checks.utils.PropertyDefaultValues;
+
+public class RulesDefinitionUtils {
+
+  private RulesDefinitionUtils() {
+  }
+
+  public static void setDefaultValuesForParameters(RulesDefinition.NewRepository repository, List<Class> checks, Language language) {
```

Comment:
This method has no test at the moment

Reply:
It is tested indirectly via Ruby and Kotlin rule definitions, but OK I will added unit test just for this one